assets/src/components/delivery/AlternativesPreferenceSelector.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useEffect, useState } from 'react';
```

Comment:
Delivery react component that allows a learner to select their alternatives preference. When a selection is made/changed, an event goes out to update all other selector of the same alternative group on the page so they are all in sync.

Alternative options are passed to the component as part of rendering (no async call)